pkg/apis/serving/v1/revision_defaults.go

```diff
@@ -158,6 +162,37 @@ func (*RevisionSpec) applyProbes(container *corev1.Container) {
 	}
 }
 
+//
+func (rs *RevisionSpec) defaultSecurityContext(ctx context.Context, container *corev1.Container, cfg *config.Config) {
+	if cfg.Features.SecurePodDefaults != config.Enabled {
+		return
+	}
+
+	if container.SecurityContext == nil {
+		container.SecurityContext = &corev1.SecurityContext{}
+	}
+
+	if container.SecurityContext.AllowPrivilegeEscalation == nil {
+		container.SecurityContext.AllowPrivilegeEscalation = ptr.Bool(false)
+	}
+	if container.SecurityContext.SeccompProfile == nil {
+		container.SecurityContext.SeccompProfile = &corev1.SeccompProfile{}
+	}
+	if container.SecurityContext.SeccompProfile.Type == "" {
+		container.SecurityContext.SeccompProfile.Type = corev1.SeccompProfileTypeRuntimeDefault
+	}
+	if container.SecurityContext.Capabilities == nil {
+		container.SecurityContext.Capabilities = &corev1.Capabilities{}
+	}
+	if container.SecurityContext.Capabilities.Drop == nil {
+		container.SecurityContext.Capabilities.Drop = []corev1.Capability{"ALL"}
+	}
+	// Default in NET_BIND_SERVICE to allow binding to low-numbered ports.
+	if container.SecurityContext.Capabilities.Add == nil {
+		container.SecurityContext.Capabilities.Add = []corev1.Capability{"NET_BIND_SERVICE"}
+	}
```

Comment:
I wonder if this will trip any naive policies that look for ~any added capabilities.  We should only need this when the user is specifically requesting a low port, which we could know from the enclosing context, but it'd take some gross plumbing to figure that out.

Reply:
Updated to do this check.